keyboards/kprepublic/bm80/readme.md

```diff
@@ -0,0 +1,19 @@
+# bm80
+
+![bm80](https://i.imgur.com/AqjEjOi.jpeg)
+
+A tkl hotswap in switch RGB keyboard from KPRepublic.
+
+* Keyboard Maintainer: [peepeetee](https://github.com/peepeetee)
+* Hardware Supported: BM80
+* Hardware Availability: [KP Republic](https://kprepublic.com/collections/new-arrival/products/bm80rgb-bm80-rgb-80-hot-swappable-custom-mechanical-keyboard-pcb-programmed-qmk-via-firmware-full-rgb-switch-underglow-type-c)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to put the keyboard into bootloader mode?   Eg, does it have a hardware reset button? Etc.

Reply:
Done, see 60549e3